index.css

```diff
@@ -247,15 +238,25 @@ section ul, section li {
 }
 
 .proposal-detail-label {
+    color: #aaa;
+    font-weight: 200;
+    letter-spacing: -0.3px;
+    text-transform: lowercase;
```

Comment:
I think we should keep the case of the labels as-is with the`:`, rather than small caps. This also lets us be consistent with the `Filtered by` label formatting in the nav.